docs/src/pages/docs/functions/combinators.md

```diff
@@ -48,6 +48,71 @@ This is a very handy dandy function, used a lot. Pass it any value and it will
 give you back a function that will return that same value no matter what you
 pass it.
 
+#### converge
+
+`crocks/combinators/converge`
+
+```haskell
+converge :: (b -> c -> d) -> (a -> b) -> (a -> c) -> a -> d
+```
+
+Provides a means of passing an acculumating function and two branching functions.
+A value can be applied to the resulting function which will then be applied to
+each branching function, the results of which will be applied to the accumulating
+function.
+
+```javascript
+import Maybe from 'crocks/Maybe'
+import alt from 'crocks/pointfree/alt'
+import converge from 'crocks/combinators/converge'
+import prop from 'crocks/Maybe/prop'
+
+const { Just } = Maybe
+
+// data :: [ Number ]
+const data = [ 1, 2, 3, 4, 5 ]
+
+// divide :: Number -> Number -> Number
+const divide = x => y => x / y
+// sum :: [ Number ] -> Number
+const sum = xs => xs.reduce((m, n) => m + n, 0)
+// length :: [ a ] -> Number
+const length = xs => xs.length
+// average :: [ Number ] -> Number
+const average = converge(divide, sum, length)
+
+average(data)
+//=> 3
+
+// maybeGetDisplay :: Object -> Maybe String
```

Comment:
`prop` can actually take any `a` because of the `Maybe`, so these should be `a -> Maybe String`

Reply:
Should this maybe be `a -> Maybe b` since the property value might not necessarily be a string?

Reply:
Good call you are right!!